src/app/settings/shelf-settings/components/edit-static-buttons-dialog/edit-static-buttons-dialog.component.html

```diff
@@ -0,0 +1,3 @@
+<sofie-sidebar-container i18n-title title="edit-static-buttons-dialog.header">
```

Comment:
We should change `title` to something else.
`i18n-title` adds a tooltip almost everywhere inside the component...
![image](https://github.com/tv2/ng-sofie/assets/23049647/b45922a9-2c81-439a-9959-99bc370f06ff)

Reply:
I have changed it to follow way it's done in other files. This fixes the issue you mention!

Reply:
What I meant for you to do was to change the _attribute_ `title` on `sofie-sidebar-container` to something else than title like we did on `sofie-page`.